### PR TITLE
Communication guide: make a tag bullet point description more universal

### PR DESCRIPTION
**Pull request by Andersson007**
Communication guide: make a tag bullet point description more universal
Relates to https://github.com/ansible-collections/collection_template/pull/73

### CODE REVIEW

**Review by samccann — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 
 * Join the Ansible forum:
     * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
```
Comment (felixfontein) [inline README.md]:
```suggestion
    * [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `YOUR TAG` tag.
```

File: docs/community_guide.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ If you encounter abusive behavior violating the Code of Conduct, please refer to
 
 * Join the Ansible forum:
     * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
```
Comment (felixfontein) [inline docs/community_guide.md]:
```suggestion
    * [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `YOUR TAG` tag.
```

**Review by felixfontein — COMMENTED** [review]
Same comments as for https://github.com/ansible-collections/collection_template/pull/73.

**Comment by Andersson007** [comment]
@samccann @felixfontein thanks for reviewing!

**Pull request merged**